### DISCUSSION TITLE
Using disabled actions in workflows

### DISCUSSION THREAD

**Discussion by ns-kkodiyan** [StackStorm/st2#5921] [Q&A] [answered]
Is it possible to use a disabled action in workflows? 

That is: in the action metadata yaml: `enabled: false`
Will I be able to use that action in an orquesta workflow definition?

Usecase: I want one root level action that does the heavy lifting and workflows can use that action in a controlled manner based on the input paramters to the workflow. However, I don't want users to access the root level action directly.

**Comment by amanda11** [comment, marked as answer]
You cannot use a disabled action in a workflow. However if you have configured RBAC then if you give your users access to the workflow, and not the action then they will have permission to run the action only as part of the workflow, not standalone.

**Reply by ns-kkodiyan** [reply]
Thank you for your response @amanda11 👍
